src/bort.f

```diff
@@ -1,75 +1,27 @@
-C> 2003-11-04  D. KEYSER  -- UNIFIED/PORTABLE FOR WRF; ADDED HISTORY
-C>                           DOCUMENTATION
-C> 2009-04-21  J. ATOR    -- USE ERRWRT
+C> @author J. Woollen
+C> @date 1998-07-08
 C>
-C> USAGE:    CALL BORT (STR)
-C>   INPUT ARGUMENT LIST:
-C>     STR      - CHARACTER*(*): ERROR MESSAGE TO BE WRITTEN VIA
-C>                SUBROUTINE ERRWRT 
+C> @param[in] STR   - character*(*): Error message
 C>
-C> REMARKS:
-C>    THIS ROUTINE CALLS:        BORT_EXIT ERRWRT
-C>    THIS ROUTINE IS CALLED BY: ADN30    ATRCPT   BVERS    CHEKSTAB
-C>                               CKTABA   CLOSMG   CMPMSG   CMSGINI
-C>                               CNVED4   COBFL    COPYBF   COPYMG
-C>                               COPYSB   CPDXMM   CPYMEM   CPYUPD
-C>                               CRBMG    CWBMG    DATEBF   DATELEN
-C>                               DRFINI   DRSTPL   DUMPBF   DXDUMP
-C>                               DXMINI   GETWIN   GETTBH   IDN30
-C>                               IFBGET   IGETNTBI IGETRFEL IGETSC
-C>                               IGETTDI  INCTAB   INVMRG   IPKM
-C>                               ISIZE    IUPVS01  IUPM     JSTNUM
-C>                               LCMGDF   LSTJPB   MAKESTAB MINIMG
-C>                               MSGINI   MSGWRT   NEMTBA   NEMTBAX
-C>                               NEMTBB   NEMTBD   NENUBD   NEVN
-C>                               NEWWIN   NMSUB    NUMMTB   NVNWIN
-C>                               NXTWIN   OPENBF   OPENMB   OPENMG
-C>                               PAD      PADMSG   PARUTG   PKB
-C>                               PKBS1    PKVS01   POSAPX   RCSTPL
-C>                               RDBFDX   RDCMPS   RDMEMM   RDMEMS
-C>                               RDMGSB   RDMSGW   RDMTBB   RDMTBD
-C>                               READDX   READERME READLC   READMG
-C>                               READNS   READSB   READS3   REWNBF
-C>                               RTRCPT   SNTBBE   SNTBDE   STATUS
-C>                               STBFDX   STDMSG   STNDRD   STNTBIA
-C>                               STRBTM   STRCPT   STSEQ    TABENT
-C>                               TABSUB   TRYBUMP  UFBCNT   UFBCPY
-C>                               UFBCUP   UFBDMP   UFBEVN   UFBGET
-C>                               UFBIN3   UFBINT   UFBINX   UFBMEM
-C>                               UFBMEX   UFBMMS   UFBMNS   UFBOVR
-C>                               UFBPOS   UFBQCD   UFBQCP   UFBREP
-C>                               UFBRMS   UFBSEQ   UFBSTP   UFBTAB
-C>                               UFBTAM   UFDUMP   UPDS3    UPFTBV
-C>                               UPTDD    USRTPL   WRCMPS   WRDESC
-C>                               WRDLEN   WRDXTB   WRITDX   WRITLC
-C>                               WRITSA   WRITSB   WTSTAT   CODFLG
-C>                               RDMTBF   INITTBF  STRTBFE  GETCFMNG
-C>                               Normally not called by any application
-C>                               programs but it could be.
```

Comment:
Great that we can now rely on doxygen to generate this list. I would not really believe it was correct, otherwise...